core/src/Kokkos_WorkGraphPolicy.hpp

```diff
@@ -0,0 +1,210 @@
+#ifndef KOKKOS_WORKGRAPHPOLICY_HPP
+#define KOKKOS_WORKGRAPHPOLICY_HPP
+
+namespace Kokkos {
+namespace Experimental {
+
+template< class ... Properties >
+class WorkGraphPolicy
+{
+public:
+  using self_type = WorkGraphPolicy<Properties ... >;
+  using traits = Kokkos::Impl::PolicyTraits<Properties ... >;
+  using index_type = typename traits::index_type;
+  using execution_space = typename traits::execution_space;
+  using work_tag = typename traits::work_tag;
+  using memory_space = typename execution_space::memory_space;
+  using graph_type = Kokkos::Experimental::Crs<index_type, execution_space, void, index_type>;
+  using member_type = index_type;
+
+  graph_type graph;
+
+  WorkGraphPolicy(graph_type arg_graph):graph(arg_graph) {}
+};
+
+namespace Impl {
```

Comment:
Discuss with @dsunder and @hcedwar the use of `Kokkos::Impl::Experimental` vs. `Kokkos::Experimental::Impl`